### DISCUSSION TITLE
🆕 GitHub Changelog Recap - November

### DISCUSSION THREAD

**Discussion by Sid-ah** [community/community#8841] [Other Feature Feedback, Questions, & Ideas]
### What was big in November?

We delivered on some key improvements that we think you’ll enjoy. Creating links in Markdown has never been easier. You can now copy a link, select the text you want to linkify, and for the final step, paste it. Et voila! Another feature we’re excited about is the light high contrast theme. It’s personally one of my favorites as it offers greater contrast between foreground and background elements. Make sure to check it out! Lastly, we launched a highly-requested feature from the community, which is to allow the reordering of tabs in the new GitHub Projects (in beta). We have a lot more to [share](https://github.blog/2021-12-13-whats-new-from-github-changelog-november-2021-recap/)!

https://user-images.githubusercontent.com/17467117/146049633-6227ebbb-d18b-4c24-ab12-1e3ca3fc7a72.mp4

**Comment by Dreamza211** [comment]
> ### What was big in November?
> We delivered on some key improvements that we think you’ll enjoy. Creating links in Markdown has never been easier. You can now copy a link, select the text you want to linkify, and for the final step, paste it. Et voila! Another feature we’re excited about is the light high contrast theme. It’s personally one of my favorites as it offers greater contrast between foreground and background elements. Make sure to check it out! Lastly, we launched a highly-requested feature from the community, which is to allow the reordering of tabs in the new GitHub Projects (in beta). We have a lot more to [share](https://github.blog/2021-12-13-whats-new-from-github-changelog-november-2021-recap/)!
> 
>  light-theme-and-issues.mp4

**Reply by Favour55** [reply]
Nice one

**Comment by Dreamza211** [comment]
#8842

**Comment by Obour-Adjei** [comment]
Creative

**Comment by Sijix44** [comment]
@@

**Comment by Trade-Help** [comment]
> > ### What was big in November?
> 
> > We delivered on some key improvements that we think you’ll enjoy. Creating links in Markdown has never been easier. You can now copy a link, select the text you want to linkify, and for the final step, paste it. Et voila! Another feature we’re excited about is the light high contrast theme. It’s personally one of my favorites as it offers greater contrast between foreground and background elements. Make sure to check it out! Lastly, we launched a highly-requested feature from the community, which is to allow the reordering of tabs in the new GitHub Projects (in beta). We have a lot more to [share](https://github.blog/2021-12-13-whats-new-from-github-changelog-november-2021-recap/)!
> 
> > 
> 
> >  light-theme-and-issues.mp4
> 
> 
> 
>

**Reply by Trade-Help** [reply]
@@

**Comment by Trade-Help** [comment]
return n if ( 0..1 ).include? n
  (fibonacci(n - 1) + fibonacci(n - 2)) #recursive calls
end
